pkg/chains/formats/slsa/attest/attest.go

```diff
@@ -74,6 +76,30 @@ func Invocation(source *v1beta1.ConfigSource, params []v1beta1.Param, paramSpecs
 	}
 
 	i.Parameters = iParams
+
+	environment := map[string]map[string]string{}
+
+	annotations := map[string]string{}
+	for name, value := range meta.GetAnnotations() {
+		// Ignore annotations that are not relevant to provenance information
+		if name == corev1.LastAppliedConfigAnnotation || strings.HasPrefix(name, "chains.tekton.dev/") {
```

Comment:
Thanks for thinking of this! This seems reasonable, but we should probably document this behavior somewhere so it doesn't catch someone by surprise.

Reply:
Good idea. Added